setup.py

```diff
@@ -10,9 +10,10 @@
     url='https://github.com/jeradM/pyxeoma',
     download_url='https://github.com/jeradM/pyxeoma/archive/1.4.2.tar.gz',
     keywords=['Xeoma'],
-    license='MIT',
     install_requires=[
         'aiohttp'
     ],
-    classifiers=[]
+    classifiers=[
+        'License :: OSI Approved :: MIT License'
+    ]
```

Comment:
This can be closed. `MIT` is a valid SPDX license identifier. It's released as part of `1.4.2`.
https://github.com/jeradM/pyxeoma/commit/19140f349719008d5a4b90cd3cf9d5cba4af0aa9

Reply:
Opened https://github.com/home-assistant/core/pull/129164 to update the version used by Home Assistant.